test/modules.js

```diff
@@ -3,20 +3,20 @@ const ALIASES = require('../aliases');
 const {addAliases} = require('@loaders.gl/core/read-file/file-aliases');
 addAliases(ALIASES);
 
-import '@loaders.gl/core/../test';
+import "../modules/core/test";
 
-import '@loaders.gl/images/../test';
+import "../modules/images/test";
 
-import '@loaders.gl/draco/../test';
-import '@loaders.gl/las/../test';
-import '@loaders.gl/obj/../test';
-import '@loaders.gl/pcd/../test';
-import '@loaders.gl/ply/../test';
+import "../modules/draco/test";
```

Comment:
Why? I am using this pattern in other repos? Does ocular's aliasing mechanism not support this?

Reply:
You cannot use relative path from an alias because it can be mapped to either `<module>/src` or `<module>/dist/es5`.

Reply:
If this is going to be a regular pattern we use, we can:
- Add aliases `<module_name>/test` for all submodules
- Move data samples into either `/modules/**/test/data` if they are module-specific or  `/test/data` if they are shared.